libs/ui/src/loading.test.tsx

```diff
@@ -1,10 +1,11 @@
 import React from 'react';
-import { render } from '@testing-library/react';
+import { render, screen } from '@testing-library/react';
 
 import { Loading } from './loading';
 
 test('Renders Loading with defaults', () => {
   const { container } = render(<Loading />);
+  screen.getByText('Loading');
   expect(container.firstChild).toMatchSnapshot();
```

Comment:
Should we remove this snapshot?

Reply:
You are probably right in that most of us don't handle changes to snapshots correctly, but I like the ease of capturing all the styles in a snapshot.

My thought is to capture defaults in a snapshot, and then variations (like fullscreen) with specific assertions. That way a change to a default value (or one affecting all tests) doesn't cause all behavior to fail, just the one snapshot. And we have more explicit checks on the variations.

I don't plan to do that for all the files here, but having up to one snapshot per component seems reasonable to me.